frame/collective/src/lib.rs

```diff
@@ -647,15 +663,16 @@ decl_module! {
 				Self::deposit_event(RawEvent::Closed(proposal_hash, yes_votes, no_votes));
 				let (proposal_weight, proposal_count) =
 					Self::do_approve_proposal(seats, voting, proposal_hash, proposal);
-				return Ok(Some(
-					T::WeightInfo::close_approved(len as u32, seats, proposal_count)
-						.saturating_add(proposal_weight)
+				return Ok((
+					Some(T::WeightInfo::close_approved(len as u32, seats, proposal_count).saturating_add(proposal_weight)),
+					Pays::No
```

Comment:
This behavior should be mentioned in the docs above

Reply:
Updated the function header Doc string ...